interfaces/Config/templates/config_rss.tmpl

```diff
@@ -401,16 +401,16 @@
                             <form action="download" method="get">
```

Comment:
You commit message says "super unclear what was happening".
It still is. Please summarize in the commit comment.